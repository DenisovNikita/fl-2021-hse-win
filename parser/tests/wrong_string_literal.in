begin function main 00
begin body
    var string s = lolkek
end body
end function