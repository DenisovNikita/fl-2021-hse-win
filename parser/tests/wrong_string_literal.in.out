Error in program.
Expect syntax: 'var string <name> = "data"', but found: 'var string <name> = lolkek'
